extensions/amp-script/0.1/test/validator-amp-script.html

```diff
@@ -35,6 +35,10 @@
     </div>
   </amp-script>
 
+  <!-- Valid: 'nodom' attribute. -->
+  <amp-script layout=container src="https://example.com/foo.js">
```

Comment:
missing `nodom` attribute in `amp-script`

Reply:
well that is embarrassing. pushing a fix now